AUTHENTICATION.md

```diff
@@ -39,14 +50,16 @@ helm repo add crossplane-alpha https://charts.crossplane.io/alpha
 helm install crossplane --namespace crossplane-system crossplane-alpha/crossplane
 ```
 
-Install `provider-aws` (assumes you have [installed the Crossplane
-CLI](https://crossplane.io/docs/v0.13/getting-started/install-configure.html#install-crossplane-cli)):
+`provider-aws` can be installed with the [Crossplane
+CLI](https://crossplane.io/docs/v0.13/getting-started/install-configure.html#install-crossplane-cli),
+but we will do so manually so that we can also create and reference a
+`ControllerConfig`:
 
 ```
-kubectl crossplane install provider crossplane/provider-aws:master
+envsubst < irsa-controller-config.yaml | kubectl apply -f -
```

Comment:
Can we have the YAML snippet (or as an addition) for doing this quickly? Otherwise, they would need to go out of this doc to find the file.

Reply:
Yeah I could add it inline. Could also update this to point to raw github URL, which might be easiest, but some folks may be averse to

Reply:
Inline YAML sounds like a good way to also let them see what's happening right here. Raw github URL might require some branch/release name in the URL and that may not be worth the hassle